### DISCUSSION TITLE
Coordinate systems

### DISCUSSION THREAD

**Discussion by aclel** [TerriaJS/terriajs#7244] [General]
Is there any way to support coordinate systems other than EPSG:4326 or EPSG:3857?

I've got some higher accuracy datasets that I would like to keep in MGA2020 Zone 55 (EPSG:7855) or in GDA2020 (EPSG:7844)

**Comment by na9da** [comment]
For 3d mode Terria uses CesiumJS which only supports EPSG:4326. There's an [open issue in CesiumJS project](https://github.com/CesiumGS/cesium/issues/3877) to support other projections.

In 2d mode, it might be possible to [add support](https://github.com/TerriaJS/terriajs/issues/6845) using Proj4Leaflet (which i think reprojects on the fly) but as of now it is not possible.
